python/swe/benchmark/run_evaluation.py

```diff
@@ -197,21 +200,72 @@ def setup_workspace(repo, repo_to_workspace_map, repo_to_image_id_map, base_comm
     )
 
 
+def check_and_pull_image(image_name):
```

Comment:
Maybe there is a better utility around it just check some library?

Reply:
will check and update in future pr